chapters/ch33.md

```diff
@@ -1,52 +1,50 @@
 > # 33. Why we compare to human-level performance
 
---> _replace THIS LINE by your translation for the above line_
+--> Tại sao chúng ta so sánh với hiệu suất của con người?
```

Comment:
Cảm ơn bạn.

Ý mình trong dòng "--> _replace THIS LINE by your translation for the above line_" là thay **cả dòng** đó, cả dấu mũi tên luôn.

Reply:
Cụm "human-level performance" được dịch từ đầu là "hiệu quả cấp con người". Mình đã tạo PR mới cho từ này ở đây https://github.com/aivivn/Machine-Learning-Yearning-Vietnamese-Translation/pull/250.

Bạn vui lòng cũng dịch theo như thế cho thống nhất. Sau này nếu tìm được từ nào hợp lý hơn thì sẽ thay toàn bộ.